docs/rst/user_manual/configuration.rst

```diff
@@ -93,6 +93,28 @@ topics and |ddsrouter| participants) are as big as to cause memory exhaustion is
 Likewise, one may choose to increase this value if wishing to deliver a greater number of samples to late joiners and
 enough memory is available.
 
+.. _user_manual_configuration_remove_unused_entities:
+
+Remove Unused Entities
+----------------------
+
+``specs`` supports a ``remove-unused-entities`` **optional** value that configures the deletion of unused internal entities in the |ddsrouter|.
+By default, unused internal entities are *not* removed.
+Thus, when the |ddsrouter| discovers a Subscriber, the |ddsrouter| creates entities in all of its participants, and these entities stay up even after the Subscriber disconnects.
+
+At times it can be useful to remove the internal entities that are not being used.
+Consider the following example.
+Two *DDS Routers* are communicating between each other through a WAN connection, when the last of their Subscribers disconnects.
+By default, the internal entities of the *DDS Routers* would *not* be removed, so the *DDS Routers* would keep consuming bandwidth, even though the data is never read.
+By setting the ``remove-unused-entities`` option to ``true``, the internal entities of the |ddsrouter| would be removed, and the *DDS Routers* would stop communicating and free up the bandwidth.
+
+.. note::
+  For the |ddsrouter| to detect that a Subscriber has disconnected, the Subscriber must be closed correctly.
+  If the Subscriber's process is killed, for example, the |ddsrouter| would not receive the disconnection signal and it would consider that the Subscriber is still running.
```

Comment:
Not so sure, we need to check if it would still work when the liveliness lost callback is raised.